test/unit/MedianOracle_gas_cost.ts

```diff
@@ -0,0 +1,49 @@
+// TODO(naguib): Fail tests if gas utilization changes
+import { ethers } from 'hardhat'
+import { BigNumber, Contract, ContractFactory, Signer } from 'ethers'
```

Comment:
I think we can just get rid of this. We wrote this when there wasn't a lot of tooling for solidity development.

We now have a hardhat plugin built in which does this out of the box ,,
https://www.npmjs.com/package/hardhat-gas-reporter